docs/design.adoc

```diff
@@ -290,6 +290,31 @@ being liquidated outside of the coverage pool.
 If both fixed amount and percentage values are set and bigger than zero,
 the fixed amount reward takes precedence.
 
+=== Auction length upgradability
+
+The default value of "Auction Length" is set during deployment of Risk Manager 
```

Comment:
s/"Auction Length"/auction length IMO, it is just a value, it is not a component or role name.